scipy/signal/filter_design.py

```diff
@@ -311,6 +312,29 @@ def zpk2tf(z, p, k):
     else:
         b = k * poly(z)
     a = atleast_1d(poly(p))
+
+    # Use real output if possible.  Copied from numpy.poly, since
```

Comment:
actually https://github.com/numpy/numpy/pull/4073 never got merged :D

so I can't say `if NumpyVersion` greater than the version it got merged in